proposals/p5448.md

```diff
@@ -0,0 +1,386 @@
+| `size_t` | >=16-bit      | `typedef uintptr_t size_t` (on most platforms) |
+|          |               | Holds the result of the `sizeof` operator.     |
+
+#### Floating-point types
+
+**Standard floating-point types**
+
+| Type          | Format                            | Width in bits    | Note                                                                        |
+| ------------- | --------------------------------- | ---------------- | --------------------------------------------------------------------------- |
+| `float`       | usually IEEE-754 binary32         | 32-bits          | The format or the size can vary depending on the compiler and the platform. |
+| `double`      | usually IEEE-754 binary64         | 64-bits          | The format or the size can vary depending on the compiler and the platform. |
+| `long double` | IEEE-754 binary128                | 128-bit          | used by some SPARC, MIPS, ARM64 implementations.                            |
+|               | IEEE-754 binary64-extended format | 80-bit or 64-bit | 80-bit (most x86 and x86-64 implementations); 64-bit used by MSVC.          |
+|               | `double-double`                   | 128-bit          | used on PowerPC.                                                            |
+
+**Fixed-width floating-point types (C++23)**
+
+They aren’t aliases to the standard floating-point types (`float`, `double`,
+`long double`), but to an extended floating-point type.
+
+| Type              | Width in bits | Defined as                     |
+| ----------------- | ------------- | ------------------------------ |
+| `std::float16_t`  | 16-bit        | `using float16_t = _Float16`   |
+| `std::float32_t`  | 32-bit        | `using float32_t = _Float32`   |
+| `std::float64_t`  | 64-bit        | `using float64_t = _Float64`   |
+| `std::float128_t` | 128-bit       | `using float128_t = _Float128` |
+| `std::bfloat16_t` | 16-bit        |                                |
+
+### Data models
+
+The following data models are widely accepted:
+
+-   32-bit systems:
+    -   `LP32` (Win16 API): `int` 16-bit; `long` 32-bit; `pointer` 32-bit.
+    -   `ILP32` (Win32 API; Unix and Unix-like systems): `int` 32-bit; `long`
+        32-bit; `pointer` 32-bit.
+-   64-bit systems:
+    -   `LLP64` (Win32 API: 64-bit ARM or x86-64): `int` 32-bit; `long` 32-bit;
+        `pointer` 64-bit.
+    -   `LP64` (Unix and Unix-like systems (Linux, macOS)): `int` 32-bit; `long`
+        64-bit; `pointer` 64-bit.
+
+[Carbon supported platforms](/docs/project/principles/success_criteria.md#modern-os-platforms-hardware-architectures-and-environments)
+
+Carbon will prioritize supporting modern OS, 64-bit little endian platforms (for
+example [LLP64](/proposals/p5448.md#data-models),
+[LP64](/proposals/p5448.md#data-models)). Historic platforms like
+[LP32](/proposals/p5448.md#data-models) won't be supported.
+
+## Proposal
```

Comment:
Can we say explicitly that if C++ considered two types to be different types, Carbon would treat them as different types as well?
I see you write it below, but you can put it above instead to clarify this guides the mapping.